test/test_blockarrays.jl

```diff
@@ -318,61 +320,6 @@ end
             @test all(isone, M)
         end
 
-        @testset "zero dim" begin
```

Comment:
Why is this removed?

Reply:
Hmm - I don't know. Wasn't intentional - I will revert.